content/projects/data-science-specific/k-means-clustering/_index.md

```diff
@@ -28,14 +28,22 @@ title: K-Means Clustering Assignment
 
 ## Pre-requisites
 
-Read through the K-Means Tutorials at {{< contentlink path="topics/data-science-specific/clustering" >}} before starting this project.
+Read through the K-Means Tutorials at [content link](http://syllabus.africacode.net/topics/data-science-specific/clustering/) before starting this project.
 
-## Clustering whisky distilleries according to tasting Profiles
+## About the DataSet
 
-Data: [Whisky Tasting Profiles](whisky.csv)
+Goal: Clustering whisky distilleries according to tasting Profiles
 
-Use K-Means clustering to cluster whisky distilleries by their tasting profile. Use the elbow or silhouette method to find the optimal number of clusters.
+Download the dataset: [Whisky Tasting Profiles](whisky.csv)
 
-To see how successful clustering was, report relevant metrics (e.g. silhouette, adjusted rand index, etc.) and create a plot showing the different distilleries, their classes according to the k-Means clustering, and the distance between points. You can use `sklearn.manifold` to get Euclidean distances between points.
+## Instructions
 
-Describe the main differences between the cluster - what are the factors that differ between classes?
\ No newline at end of file
+1. Use K-Means clustering to cluster whisky distilleries by their tasting profile. Use the elbow or silhouette method to find the optimal number of clusters.
```

Comment:
why is K-Means capitalised like that? It's normal to talk about k-means clustering in normal sentences without fancy capitalisation.

Reply:
K-Means is often capitalized in that way because it is the name of a specific clustering algorithm, and it's a common convention in the field of computer science and machine learning to capitalize the names of algorithms and technologies to distinguish them as proper nouns. This practice helps make it clear that you are referring to a specific algorithm and not using the term "k-means" as a general description.